### PR TITLE
Validate Extensions (easy)

### PR DESCRIPTION
**Pull request by atlas0fd00m**
validate each module in the VIV_EXT_PATH and VDB_EXT_PATH is a valid module, with a `vivExtension()` or `vdbExtension()` function.

log skips and loaded modules

### CODE REVIEW

File: vdb/extensions/__init__.py [hunk 1]
```diff
@@ -57,8 +60,15 @@ def loadExtensions(vdb, trace):
                 module.vdb = vdb
                 module.__file__ = modpath
                 spec.loader.exec_module(module)
+
+                if not hasattr(module, 'vdbExtension'):
+                    logger.info("Skipping python module %r, not a valid extension", modpath)
```
Comment (invisig0th) [inline vdb/extensions/__init__.py:65]: Could we make it a bit more explicit like `module has no vdbExtension() function` here and below? We've found a bit of spoon feeding in log messages goes a long ways :)

File: vivisect/extensions/__init__.py [hunk 2]
```diff
@@ -52,8 +54,15 @@ def loadExtensions(vw, vwgui):
                 module.vw = vw
                 spec.loader.exec_module(module)
 
+                if not hasattr(module, 'vivExtension'):
+                    logger.info("Skipping python module %r, not a valid extension", modpath)
```
Comment (invisig0th) [inline vivisect/extensions/__init__.py:58]: Seems like making these into `warning()` calls would be appropriate. If you've got things in your modules path that don't implement the convention :D

File: vdb/extensions/__init__.py (hunk 1 above)
Comment (atlas0fd00m) [reply vdb/extensions/__init__.py:65]:
```suggestion
                    logger.warning("Skipping python module %r, not a valid extension (module has no vdbExtension() function)", modpath)
```

File: vivisect/extensions/__init__.py (hunk 2 above)
Comment (atlas0fd00m) [reply vivisect/extensions/__init__.py:58]:
```suggestion
                    logger.warning("Skipping python module %r, not a valid extension (module has no vivExtension() function)", modpath)
```

**Review by invisig0th — APPROVED** [review]

**Pull request merged**